subworkflows/nf-core/abundance_differential_filter/meta.yml

```diff
@@ -122,5 +132,8 @@ authors:
   - "@bjlang"
   - "@araiz2001"
   - "@suzannejin"
+  - "@roskamsh"
+  - "@syangaskhan"
+  - "@WackerO"
```

Comment:
Maybe add yourself as a maintainer?

Did all these authors contribute to your version of the differential subworkflow?

Reply:
The last three they actually helped during the hackathon to call the multi-channel logic on DESeq2 and limma.
I'm not exactly sure up to which point the contributions should be added as author here in the meta, but wondering that it is anyway nice to credit them